src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -113,12 +116,23 @@
 (rf/reg-event-fx :wallet/add-authorized-transaction
  (fn [{:keys [db]} [transaction]]
    (let [transaction-hashes (:hashes transaction)
-         chain-id           (key (first transaction-hashes))
-         tx-id              (first (val (first transaction-hashes)))
-         transaction-detes  {:status   :pending
-                             :id       (:id transaction)
-                             :chain-id chain-id}]
-     {:db (assoc-in db [:wallet :transactions tx-id] transaction-detes)
+         tx-ids             (flatten (map (fn [[_key value]] value) transaction-hashes))
+         tx-id-with-details (reduce-kv (fn [m1 chain-id v1]
+                                         (merge m1
```

Comment:
see the wallet subs below to see this structure.
Perhaps it needs some comment or something to explain why we want it this way.

Reply:
What is `tx` `m1` `v1`?

Reply:
tx = transaction. I can change to transaction instead. 
m1 is map1 and v1 value1. I think the best thing I can do is extract this to a helper function and have a test to define the input and output 👍

Reply:
That would be better

Reply:
done